tests/e2e/src/steps/portal.steps.ts

```diff
@@ -226,6 +226,14 @@ Given("I am on the Main page", async function (this: ICustomWorld) {
   await expect(this.page?.url()).toBe(config.BASE_URL + NetworkSwitcher.zkSyncEraGoerli);
 });
 
+Given("I am on the Main page without a NetworkSwitcher", async function (this: ICustomWorld) {
```

Comment:
Given I am on main page - contains NetworkSwitcher.zkSyncEraGoerli that fails our step as in the end we have just config.BASE_URL in the URL. 
How often are we gonna face a necessity to use it further? - here is a first usage, in test cases we will use that same approach at least for Withdraw and Transfer. Ideally it will work after any transaction at the end. 

We also need a strict verification that we have at the very beginning, that generally Given I am on main page do. I thought should we modify Given I am on main page to make it more universal or do not do overengineering. I choose 2nd option at this case.